graphql/schema.graphql

```diff
@@ -3,9 +3,18 @@ type Query {
   patch(id: String!): Patch!
   task(taskId: String!): Task
   projects: Projects!
+  patchTasks(
+    patchId: String!
+    sortBy: TaskSortCategory = STATUS
+    sortDirection: SortDirection = ASC
+    page: Int = 0
+    limit: Int = 0
+    searchInput: String = ""
+    statuses: [String] = []
```

Comment:
I think we want this to be [String!]! both to avoid handling null values in the array and to avoid handling the array being null (since you're defaulting it anyway). Similarly with the other params here - make them non-nullable if appropriate so that you don't have to handle null values in the resolver